### ISSUE TITLE
Filtered framewise displacement

### ISSUE THREAD

**Issue by rciric** [poldracklab/fmriprep#2094]
Respiratory processes introduce high-frequency “pseudo-motions”; the image is factitiously shifted (principally in the phase encoding direction) as changes in abdominal volume perturb B0. To account for pseudo-motions, it is suggested that we use a filtered version of framewise displacement that removes the high frequencies where pseudo-motion is typically most apparent.

Following work from [Gratton and colleagues](https://www.biorxiv.org/content/10.1101/837161v1.full.pdf), filtered framewise displacement is computed as follows:
(1) padding of motion parameter time series
(2) low-pass filtering of motion parameter time series at 0.1 Hz using a first-order Butterworth filter
(3) computation of FD following the Power definition (sub of absolute differences)
While computing filtered Power FD is straightforward, filtered Jenkinson FD would require additional considerations to implement.

**Comment by tsalo** [comment]
I think this can be left to post-processing workflows (e.g., an fMRIPost App or, if I'm going to self-promote, XCP-D). The Gratton paper was specific to single-band data, while the multiband version from [Fair et al. (2020)](https://www.sciencedirect.com/science/article/pii/S1053811919309917) recommends a notch filter with parameters selected based on either direct inspection of the data, or at least based on participant age, and would require adding more parameters to fMRIPrep. Since FD would be used for censoring and denoising in a post-processing step, it seems like the filtering can be done there as well.

**Comment by effigies** [comment]
Is filtering motion parameters purely for denoising, or should it be applied to the parameters for motion correcting?

**Comment by tsalo** [comment]
I believe it's just for the denoising, and for flagging high-motion volumes to censor. @madisoth does that sound right?

**Comment by madisoth** [comment]
Right, just for denoising and motion censoring in postprocessing (e.g. XCP-D, DCANBOLDProcessing)

**Comment by tsalo** [comment]
Thanks @madisoth! I think it's safe to consider this a postprocessing concern then.

**Comment by effigies** [comment]
Might be good to come up with a column description field to indicate whether the motion parameter has already been filtered, to avoid multiple application.
